### ISSUE TITLE
Implement Data Transforms.

### ISSUE THREAD

**Issue by hasanemirakin** [odtuyzt/time-series-analysis#13]
Implement data transformations functions such as power transforms, standardization and normalizations. Priorities are:

-Box-Cox (https://en.wikipedia.org/wiki/Power_transform#Box%E2%80%93Cox_transformation)
-Standardization (https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html)
-Normalization (https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html)

Please share if you have any other methods in mind.

**Comment by hasanemirakin** [comment]
I will implement these transformations

**Comment by hasanemirakin** [comment]
box_cox is implemented, working on standardization and normalization

**Comment by hasanemirakin** [comment]
standardization and minmaxscaling was implemented
